src/create-release.js

```diff
@@ -19,6 +20,16 @@ async function run() {
     const draft = core.getInput('draft', { required: false }) === 'true';
     const prerelease = core.getInput('prerelease', { required: false }) === 'true';
 
+    const bodyPath = core.getInput('body_path', { required: false });
+    let bodyFileContent = null;
+    if (bodyPath !== '' && !!bodyPath) {
```

Comment:
I won't speak to the style that the actions team prefers, but `bodyPath !== '' && ` is redundant, here. (The boolean value of the empty string is `false`.) Which would allow further reduction to just `if(bodyPath)`. There's no reason to force cast to an actual boolean here because the type casting is already handled by falsiness.